onebusaway-android/src/main/res/layout/fragment_trip_plan_results.xml

```diff
@@ -0,0 +1,273 @@
+                    android:ellipsize="end"/>
+
+            <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAppearance="?android:attr/textAppearanceMedium"
+                    android:text="Medium Text"
+                    android:id="@+id/option2Duration"
+                    android:textColor="@color/body_text_2"
+                    android:textSize="15sp"/>
+
+        </LinearLayout>
+
+        <LinearLayout
+                android:orientation="vertical"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_weight="1"
+                android:background="@color/trip_option_background"
+                android:gravity="center_vertical"
+                android:id="@+id/option3LinearLayout"
+                android:paddingLeft="5dp"
+                android:paddingRight="5dp">
+
+            <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAppearance="?android:attr/textAppearanceMedium"
+                    android:id="@+id/option3Title"
+                    android:textColor="@color/body_text_2"
+                    android:singleLine="true"
+                    android:ellipsize="end"
+            />
+
+            <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAppearance="?android:attr/textAppearanceMedium"
+                    android:text="Medium Text"
+                    android:id="@+id/option3Duration"
+                    android:textColor="@color/body_text_2"
+                    android:textSize="15sp"/>
+
+        </LinearLayout>
+    </LinearLayout>
+
+
+    <LinearLayout
+            android:orientation="horizontal"
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"
+            android:id="@+id/switchViewLayout"
+            android:layout_marginTop="2dp"
+            android:background="@android:color/white"
+            android:layout_marginBottom="2dp">
+
+        <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:textAppearance="?android:attr/textAppearanceMedium"
+                android:text="@string/trip_plan_switch_view"
+                android:id="@+id/textView2"
+                android:gravity="center_vertical"
+                android:layout_marginLeft="10dp"/>
+
+        <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:id="@+id/switchViewImageView"
+                android:layout_gravity="center_horizontal"
+                android:src="@drawable/ic_action_map"/>
```

Comment:
We already have `@drawable/ic_arrivals_styleb_action_map`, which is the same icon (but black and dynamically colored) - you should be able to use this existing icon rather than including another.